src/anim/controller/anim-state.js

```diff
@@ -75,12 +105,12 @@ class AnimState {
     }
 
     get nodeCount() {
-        if (!this._blendTree || !(this._blendTree.constructor === AnimBlendTree)) return 1;
+        if (!this._blendTree || (this._blendTree.constructor === AnimNode)) return 1;
         return this._blendTree.getNodeCount();
     }
 
     get playable() {
-        return (this.name === ANIM_STATE_START || this.name === ANIM_STATE_END || this.name === ANIM_STATE_ANY || this.animations.length === this.nodeCount);
+        return (ANIM_CONTROL_STATES.includes(this.name) || this.animations.length === this.nodeCount);
```

Comment:
`Array#includes` doesn't exist in IE11. Have you checked the Babel output to see if it's IE11-compatible?